velero/backup/common-service/env.properties

```diff
@@ -1,13 +1,13 @@
 # Change the following values to match your environment
-OPERATOR_NS="" # Pass the namespace where the cs operator is installed
+OPERATOR_NS="test-br" # Pass the namespace where the cs operator is installed
 
 # Pass the namespace where the cs services are installed
 # Leave it as empty string if the services are installed in the same namespace as the operator
-SERVICES_NS="" 
+SERVICES_NS="test-br" 
 CONTROL_NS="" # Pass the control namespace if it is needed to be backed up
 
 #Pass any additional namespaces in the tenant that are not the operator or services namespace. Comma delimited
-TETHERED_NS=""
+TETHERED_NS="tenant1,tenant2"
```

Comment:
Make sure to remove these test values before merging

Reply:
Oh, Good catch, thanks, Ben.